src/network/NetworkAction.cpp

```diff
@@ -191,7 +191,7 @@ const std::vector<NetworkAction> NetworkActions::Actions =
     }, {
         STR_ACTION_KICK_PLAYER, "PERMISSION_KICK_PLAYER",
         {
-            GAME_COMMAND_KICK_PLAYER
+            -4
```

Comment:
@wolfreak100 I mean the number itself. But I just checked and saw there were `-1`, `-2`, and `-3` further in the list as well. It just seems odd to me someone would replace a macro with a magic number.